ludwig/features/image_feature.py

```diff
@@ -90,6 +92,31 @@
 ###
 # Image specific augmentation operations
 ###
+@register_augmentation_op(name="auto_augmentation", features=IMAGE)
+class AutoAugment(torch.nn.Module):
+    def __init__(self, config: AutoAugmentationConfig):
+        super().__init__()
+        self.auto_augmentation_method = config.method
+        self.augmentation_method = self.get_augmentation_method()
+
+    def get_augmentation_method(self):
+        if self.auto_augmentation_method == "trivial_augment":
+            return transforms.TrivialAugmentWide()
+        elif self.auto_augmentation_method == "auto_augment":
+            return transforms.AutoAugment()
+        elif self.auto_augmentation_method == "rand_augment":
+            return transforms.RandAugment()
+        else:
+            raise ValueError(f"Unsupported auto-augmentation method: {self.auto_augmentation_method}")
+
+    def forward(self, imgs):
```

Comment:
Is the first dimension of `imgs` the batch dimension?

Reply:
Yes.

Reply:
nit: It would be helpful to add a type hint for the `imgs` argument being passed into the forward function